bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/AnomalousCouplings-SMP/WWTolnulnu_01j_aTGC_lep_WWmass_4f_NLO_FXFX_UL/README.md

```diff
@@ -0,0 +1,81 @@
+Gridpacks production: UL WW aTGC via cmsconnect
+======================
+
+### 0 - Set up
+
+Get a CMS connect account: https://twiki.cern.ch/twiki/bin/view/CMSPublic/WorkBookCMSConnect#Sign_up_to_CMS_Connect
+
+    ssh -Y <username>@login-el7.uscms.org -o ServerAliveInterval=240
+
+    cmsrel CMSSW_9_3_16 
```

Comment:
Why do you set up this?

Reply:
Because CMSconnect allowed us to use the slc6 architecture